onnxruntime/test/optimizer/transpose_optimizer_test.cc

```diff
@@ -294,6 +294,17 @@ TEST(TransposeOptimizerTests, TestPadNonconst) {
                     /*opset_version*/ 11);
 }
 
+// The CUDA Resize kernel assumes that the input is NCHW and
+// Resize can't be supported in ORT builds with CUDA enabled.
+// TODO: Enable this once the CUDA Resize kernel is implemented
+// "generically" (i.e.) aligning with the generic nature of the
+// ONNX spec.
+// See https://github.com/microsoft/onnxruntime/pull/10824 for
+// a similar fix applied to the CPU Resize kernel.
+// Per tests included in #10824, the ROCM EP also generates
+// incorrect results when this handler is used, so the Resize
+// handler is not enabled even for those builds.
+#if !defined(USE_CUDA) && !defined(USE_ROCM)
```

Comment:
Is it possible to give a warning or skip instead just comment out the tests for CUDA or ROCM?

Reply:
Given that the resize handler is not part of such builds (CUDA or ROCM), we will have to skip it (can't even continue with warning). But I am curious what value will it ahead instead of this ? Can you elaborate please ?